pygerrit/ssh.py

```diff
@@ -113,6 +114,7 @@ def _do_connect(self):
         if self.username is None or self.port is None:
             self._configure()
         try:
+            self.set_missing_host_key_policy(paramiko.AutoAddPolicy())
```

Comment:
The `paramiko.` prefix can be removed if `AutoAddPolicy` is imported as mentioned above.